generators/app/templates/src/config/i18n.js

```diff
@@ -1,19 +1,12 @@
 import i18next from 'i18next';
 import { initReactI18next } from 'react-i18next';
 
-function requireAll(requireContext) {
-  return requireContext.keys().map(requireContext);
-}
-
 i18next.use(initReactI18next).init({
   lng: 'es',
-  // uncomment next line if you have multilang in your application and you have a fallback language
-  // fallbackLng: 'en',
+  supportedLngs: ['es', 'en'],
+  // Comment next line if your application does not have multilang support
```

Comment:
```suggestion
  // Remove next line if your application does not have multilang support
```
We shouldn't promote commented code 😅